README.md

```diff
@@ -2,7 +2,76 @@
+- **Authentication**
+  - The AWS Elasticsearch implementation uses signed HTTP requests.
+  - The legacy service used basic authentication so you would add the `username` and `password` credentials from the service to create the authorization token in a request (ie. `curl --user <USERNAME>:<PASSWORD> -XPUT 'localhost:9200/idx'`)
+- **AWS Region**
+  - *Only applicable to the new AWS Elasticsearch service*, you will _always_ be required to include the AWS region of `us-gov-west-1` when creating a signed HTTP request. It will *always* be `us-gov-west-1`.  You can see the region in the service's `host` and `uri` credentials ie (`search-cg-broker-fake-service-host.us-gov-west-1.es.amazonaws.com`)
+
+## Examples
+
+The following is a list of different language examples for connecting and using the updated Elasticsearch service
+offering which leverages AWS Elasticsearch.
+
+### Creating an instance
+
+The following example compares the difference between creating an elasticsearch instance between the old and new service offerings.
+
+```bash
+##
+## The following examples are creating a development instance
+##
+
+### The old way
+$ cf create-service elasticsearch56 medium dev-elastic-service
+
+### The new way
+$ cf create-service aws-elasticsearch aws-dev dev-elastic-service
+
+
+##
+## The following examples are creating a production, high availability instance
+##
+
+### The old way
+$ cf create-service elasticsearch56 medium-ha prod-elastic-service
+
+### The new way
+$ cf create-service aws-elasticsearch aws-standard prod-selastic-service
+```
+
+### Interacting with an instance
+
+- See the [./python](./python) example for connecting, loading, and querying data using the Python language.
+
+## Additional AWS Elasticsearch resources
+
+- [What Is Amazon Elasticsearch Service?](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/what-is-amazon-elasticsearch-service.html)
+- [Signing HTTP Requests to Amazon Elasticsearch Service](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-request-signing.html)
+  - Language examples:
+    - [Java](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-request-signing.html#es-request-signing-java)
+    - [Python](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-request-signing.html#es-request-signing-python)
+    - [Ruby](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-request-signing.html#es-request-signing-ruby)
+    - [Node](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-request-signing.html#es-request-signing-node)
+    - [Go](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-request-signing.html#es-request-signing-go)
+- The cloud.gov [aws-broker](https://github.com/cloud-gov/aws-broker) provides the underlying marketplace service
```

Comment:
Awesome, this is nice to see and should be helpful for customers who have additional questions or want to dive deeper!